motorway/contrib/amazon_sqs/utils.py

```diff
@@ -1,10 +1,17 @@
 import json
 from boto.sqs.message import Message
+from motorway.contrib.amazon_sqs.exceptions import SQSMessageToLarge
+
+SQS_MESSAGE_MAXIMUM_SIZE = 262144
```

Comment:
Add a link to the SQS documentation stating this.

Reply:
fixed